controller/engine_image_controller.go

```diff
@@ -654,8 +661,9 @@ func (ic *EngineImageController) enqueueControlleeChange(obj interface{}) {
 	}
 
 	metaObj, err := meta.Accessor(obj)
+	log := ic.logger.WithField("engineImage", obj)
```

Comment:
Don't use this, since you are throwing in the whole crd object into the field.
Better to just use `ic.logger.WithError(err).Warn(.......)` since it's not know what the actual obj will represent and by using the ic.logger we still get controller + node fields.

See below suggestion by @jenting